lib/nerves_hub_link/configurator.ex

```diff
@@ -71,4 +73,9 @@ defmodule NervesHubLink.Configurator do
       Default
     end
   end
+
+  defp fwup_version do
+    {version_string, 0} = System.cmd("fwup", ["--version"])
+    version_string |> String.trim()
```

Comment:
```suggestion
String.trim(version_string)
```
piping for a short call might be unnecessary